dietpi-install.sh

```diff
@@ -32,6 +35,10 @@ qm set "$ID" --net0 'virtio,bridge=vmbr0'
 qm set "$ID" --scsi0 "$STORAGE:vm-$ID-disk-0"
 qm set "$ID" --boot order='scsi0'
 qm set "$ID" --scsihw virtio-scsi-pci
+qm set "$ID" --name "dietpi" >/dev/null
+qm set "$ID" --description "### [DietPi](https://dietpi.com)
+### [DietPi Forums](https://dietpi.com/forum)
+### [DietPi Software](https://dietpi.com/docs/software/)" >/dev/null
```

Comment:
Suggesting to stick with single-quotes and for completeness list all 4 parts of the DietPi website:
```suggestion
qm set "$ID" --name 'dietpi' >/dev/null
qm set "$ID" --description '### [DietPi Website](https://dietpi.com/)
### [DietPi Docs](https://dietpi.com/docs/)
### [DietPi Forum](https://dietpi.com/forum/)
### [DietPi Blog](https://dietpi.com/blog/)' >/dev/null
```
I currently don't know where/how this description is shown. There is no problem with 4 lines and links with this Markdown syntax are nicely expanded?

Reply:
The description is shown in the Proxmox GUI when the VM is created.

https://gyazo.com/43661c9492152472d196222478315459